File: Python/car.py
# Importamos las clases
from driver import Driver

# Clase carro
class Car:

    # Atributos o propiedades de la clase
    id = int
    license = str
    driver = Driver("","")
    passenger = str

    # Método constructor
    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

File: PHP/Car.php
<?php
//Importación de clases
require_once('Account.php');

//Clase carro
class Car
{
    // Atributos o propiedades de la clase
    public $id;
    public $license;
    public $driver;
    public $passenger;

    //Método constructor
    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    //Método para visualizar los datos del carro
    public function showDataCar()
    {
        echo "License: $this->license, Driver: ".$this->driver->name;
    }
}
